rehlds/public/rehlds/static_map.h

```diff
@@ -229,8 +229,8 @@ class CICaseStringKeyStaticMap : public CStaticMap<const char*, T_VAL, ASSOC_2N,
 		if (cpuinfo.sse4_2) {
 			while (*pcc) {
 				char cc = *(pcc++);
-				if (cc >= 'A' || cc <= 'Z') {
-					cc |= 0x20;
+				if (cc >= 'A' && cc <= 'Z') {
+					cc += ('a' - 'A');

```

Comment:
IMO that is more readable:
```suggestion
					cc = (cc - 'A') + 'a';
```

Reply:
I would like to use Q_strlwr but....

Reply:
I would like to use tolower() but....

Thanks. Done b7809bd